<launch>
  <node pkg="final" 
        type="bb_img_publisher_node.py" 
        name="bb_img_publisher_node" 
        output="screen"/>
</launch>
